pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -1008,6 +1008,12 @@ func (r *Builder) ensureVolumePopulatorPVC(workload *model.Workload, image *mode
 			originalVolumeDiskId = imageProperty.(string)
 		}
 
+		// Check if there's a storage map available
+		if len(r.Context.Map.Storage.Spec.Map) == 0 {
```

Comment:
how did we reach to this state at all? the user created the plan without storage mapping?

Reply:
yes, it's possible to do that when the VM only has an image attached (vs. having a volume attached)